src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CaseConfigurationProviderServiceTest.java

```diff
@@ -1075,6 +1077,136 @@ void should_not_recalculate_due_date_interval_from_given_due_date_origin_propert
         Assertions.assertThat(mappedData.getConfigurationDmnResponse()).isEmpty();
     }
 
+    @Test
+    void should_not_include_single_db_additional_properties_for_camunda_add_properties_on_reconfigure_request() {
+        String someCaseId = "someCaseId";
+        String taskAttributesString = "{\"taskType\":\"taskType\"}";
+        Map<String, Object> taskAttributes = Map.of("taskType", "taskType");
+
+        when(ccdDataService.getCaseData(someCaseId)).thenReturn(caseDetails);
+        when(caseDetails.getData()).thenReturn(Map.of("caseAccessCategory", ""));
+
+        List<PermissionsDmnEvaluationResponse> permissions = asList(
+                new PermissionsDmnEvaluationResponse(
+                        stringValue("tribunalCaseworker"),
+                        stringValue("Read,Refer,Own,Manage,Cancel"),
+                        null,
+                        null,
+                        null,
+                        stringValue("LEGAL_OPERATIONS"),
+                        stringValue("categoryB")
+                ),
+                new PermissionsDmnEvaluationResponse(
+                        stringValue("seniorTribunalCaseworker"),
+                        stringValue("Read,Refer,Own,Manage,Cancel"),
+                        null,
+                        null,
+                        null,
+                        stringValue("LEGAL_OPERATIONS"),
+                        stringValue("categoryC")
+                )
+        );
+
+        String caseData = "{\"caseAccessCategory\":\"\"}";
+
+        when(dmnEvaluationService.evaluateTaskPermissionsDmn("IA", "Asylum", caseData, taskAttributesString))
+                .thenReturn(permissions);
+
+        when(dmnEvaluationService.evaluateTaskConfigurationDmn("IA", "Asylum", caseData, taskAttributesString))
+                .thenReturn(asList(
+                        new ConfigurationDmnEvaluationResponse(stringValue("name1"), stringValue("value1")),
+                        new ConfigurationDmnEvaluationResponse(stringValue("name2"), stringValue("value2")),
+                        new ConfigurationDmnEvaluationResponse(
+                                stringValue("additionalProperties_name3"),
+                                stringValue("value3")
+                        ),
+                        new ConfigurationDmnEvaluationResponse(
+                                stringValue("additionalProperties_name4"),
+                                stringValue("value4")
+                        ),
+                        new ConfigurationDmnEvaluationResponse(
+                                stringValue("additionalProperties_name5"),
+                                stringValue("value5")
+                        ),
+                        new ConfigurationDmnEvaluationResponse(
+                            stringValue("additionalProperties_name6"), stringValue("value6"))
+                ));
```

Comment:
So if you're not setting the canReconfigure here I assume the default is `null` or false?  So I would expect none of them to be reconfigured unless I am missing something?